modules/apps/asset/asset-categories-admin-web/npmscripts.config.js

```diff
@@ -19,8 +19,7 @@ module.exports = {
 			'<inputDir>/js/AssetCategoriesManagementToolbarDefaultEventHandler.es.js',
 			'<inputDir>/js/AssetCategoriesSelectorTag.es.js',
 		],
+		mode: 'default',
```

Comment:
Why do we need to opt-in to the `'default'`? Shouldn't that be the default?

Reply:
Not really. For now, the default is to build with federation 'disabled' (per the defaults at the root `npmscripts.config.js`). Once we finish the webpack migration for all projects, we remove that `disabled` and these `default`s and everything builds as `default`.